### ISSUE TITLE
.form-control-static not in alignment when used in form with .form-group-sm

### ISSUE THREAD

**Issue by limitstudios** [twbs/bootstrap#16385]
.form-control-static not in alignment when used in form with .form-group-sm

Attached testcase here, tested using Firefox 37.0.2 on Windows 8.1

http://jsbin.com/copeyogela

**Comment by twbs-lmvtfy** [comment]
Hi @limitstudios!

You appear to have posted a live example (http://jsbin.com/nanipuzayu/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 18, column 3: [E029](https://github.com/twbs/bootlint/wiki/E029): Since grid classes apply to devices with screen widths greater than or equal to the breakpoint sizes (unless overridden by grid classes targeting larger screens), `class="col-xs-12 col-sm-12"` is redundant and can be simplified to `class="col-xs-12"`
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by limitstudios** [comment]
Fixed test case http://jsbin.com/copeyogela

**Comment by twbs-lmvtfy** [comment]
Hi @limitstudios!

You appear to have posted a live example (http://jsbin.com/copeyogela/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 19, column 3: [E029](https://github.com/twbs/bootlint/wiki/E029): Since grid classes apply to devices with screen widths greater than or equal to the breakpoint sizes (unless overridden by grid classes targeting larger screens), `class="col-xs-12 col-sm-12"` is redundant and can be simplified to `class="col-xs-12"`
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #15536.
